Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx

```diff
@@ -128,7 +128,7 @@ N4(int argc, char * argv[])
   ImagePointer inputImage = reader->GetOutput();
   inputImage->DisconnectPipeline();
 
-  // handle the mask image
+  // Handle the mask image
```

Comment:
in commit message: liable to throw **and** exception -> liable to throw **an** exception

Reply:
Changed in 155b729.